backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -272,6 +280,36 @@ public Schema dropCreateSchema(String name) {
     return this.schemaInfos;
   }
 
+  @Override
+  public Collection<Setting> getSettings() {
+    if (this.settings.isEmpty()) {
+      this.settings = MetadataUtils.loadSettings(jooq);
+    }
+    return this.settings;
+  }
+
+  @Override
+  public Setting createSetting(String key, String value) {
```

Comment:
actually this is saveSetting?

Reply:
it creates a setting record and saves it in the db :)